README.md

```diff
@@ -14,9 +14,9 @@
 
 - **NOTE:** OTEL can be setup to send APM data to Splunk APM, Spunk Enterprise HEC, Splunk Log Observer, or all three! Config below is for sending to all three.
```

Comment:
Does this send to Log Observer also? Or should this be just APM and Hec now given you removed the need for log observer? If so changes this line to fit current reality.

Reply:
👍